listenbrainz/webserver/static/js/jsx/spotify-player.jsx

```diff
@@ -142,6 +150,15 @@ export class SpotifyPlayer extends React.Component {
     this.setState({ errorMessage: error });
   }
 
+  handleAccountError(error) {
+    const errorMessage = 'Failed to validate Spotify premium account';
```

Comment:
Is this function only called when validating premium accounts? 401s and 403s might be due to different reasons?

Reply:
I modified the error message, removing 'premium'.
403 is for non-premium users, 401 if the spotify token is missing or invalid.
Both should result in the same action and trigger a fallback to embedded iframe